ruby/ruby_trunk/ruby_benchmarks/runner

```diff
@@ -54,3 +54,14 @@ if [ "$OPTCARROT_BENCHMARK" = true ]; then
 else
   echo "Skipping optcarrot benchmarks"
 fi
+
+if [ "$LIQUID_BENCHMARK" = true ]; then
```

Comment:
I know it's outside of the scope of your PR, but right now `ruby/ruby_trunk/ruby_benchmarks/runner` and `ruby/ruby_releases/ruby_benchmarks/runner` have a lot of duplication. The only difference is in first 20 lines of `ruby/ruby_trunk/ruby_benchmarks/runner`.

Do you think you can move the common part into a shared bash file and reduce the duplication? Then you'd only need to change one file to add Liquid part.